deploy-prod/deploy.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+echo "Kill all the running PM2 actions"
+pm2 kill
+
+echo "Jump to app folder"
+cd $HOME/LANE/LANE-client
+pwd
+
+git pull
+
+echo "Reinstalling dependencies"
+rm -rf node_modules
+npm ci
```

Comment:
I think `npm ci` automatically removes the existing `node_modules` directory. L.13 may be necessary.

Reply:
Ah I see, yes I have confirmed that this is the case. Removed the `rm -rf`